package.json

```
@@ -338,7 +339,7 @@
     "@metamask/obs-store": "^9.0.0",
     "@metamask/permission-controller": "^10.0.0",
     "@metamask/permission-log-controller": "^2.0.1",
-    "@metamask/phishing-controller": "^9.0.3",
+    "@metamask/phishing-controller": "^10.0.0",
```

You can also remove the resolutions entry above and apply the preview build here instead:

```suggestion
    "@metamask/phishing-controller": "npm:@metamask-previews/phishing-controller@11.0.0-preview-3b4ca958",
```

The resolutions entry is useful if `phishing-controller` is being used in many places in the dependency tree, and they all need to be aligned. Since `phishing-controller` does not have any downstream packages other than the clients, either way should work.